blueprints/add_drink.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+from flask import Blueprint, session, render_template
+from bson import ObjectId
+
+from .utils import *
+from .api import get_ingredients_by_type, get_flavors
+
+add_drink = Blueprint('add_drink', __name__)
+
+
+@add_drink.route('/add_cocktail/', methods=["GET", "POST"])
+def new_drink():
+    """
+    render the form for adding a new cocktail to the database
+    """
+    if session['username']:
+        return render_template('addcocktail.html')
+
+    else:
+        return redirect(url_for("index"))
+
+
+@add_drink.route('/cocktail_processing/', methods=["POST"])
+def add_new_drink_to_db():
+    """
+    function called from the cocktail form to add a drink to the
+    database using AJAX
+    """
+    ingredients = json.loads(get_ingredients_by_type(None))
```

Comment:
you shouldn't need to pass in None since it's the default argument to the function